mito-ai/src/icons/GarbageIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const GarbageIcon: React.FC = () => (
+    <svg width="14" height="14" viewBox="0 0 10 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <g fill='currentColor'>
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M9.1364 4.53183V8.67853C9.1364 9.04282 8.98838 9.373 8.74865 9.61227C8.50935 9.85157 8.17916 10 7.81487 10H2.1849C1.82059 10 1.4904 9.85198 1.25112 9.61227C1.01181 9.37297 0.863369 9.0428 0.863369 8.67853V4.53183C0.863369 4.42452 0.950952 4.33652 1.05869 4.33652H8.94079C9.04852 4.33652 9.13611 4.4241 9.13611 4.53183H9.1364ZM1.32141 1.75772H2.80914V1.32147C2.80914 0.957177 2.95758 0.627004 3.19689 0.387732C3.4362 0.14846 3.76639 0 4.13068 0H5.8692C6.23351 0 6.5637 0.148433 6.80298 0.387732C7.04227 0.627031 7.19073 0.957203 7.19073 1.32147V1.75772H8.67847C9.04278 1.75772 9.37297 1.90615 9.61225 2.14504C9.85156 2.38434 10 2.71451 10 3.07878V3.33617C10 3.57259 9.80797 3.76461 9.57154 3.76461L0.428461 3.76502C0.192028 3.76502 0 3.573 0 3.33658V3.07919C0 2.71489 0.14844 2.38472 0.38775 2.14545C0.627059 1.90618 0.957247 1.75813 1.32153 1.75813L1.32141 1.75772ZM5.86918 0.857224H4.13065C4.00318 0.857224 3.88723 0.909443 3.80293 0.993728C3.71864 1.07802 3.66642 1.19397 3.66642 1.32143V1.75768H6.33382V1.32143C6.33382 1.19397 6.2816 1.07801 6.19731 0.993728C6.11302 0.909438 5.99706 0.857224 5.8696 0.857224H5.86918ZM3.67266 5.93643C3.67266 5.70001 3.48063 5.50799 3.2442 5.50799C3.00777 5.50799 2.81574 5.70001 2.81574 5.93643V8.13416C2.81574 8.37058 3.00777 8.5626 3.2442 8.5626C3.48105 8.5626 3.67266 8.37058 3.67266 8.13416V5.93643ZM5.42847 5.93643C5.42847 5.70001 5.23644 5.50799 5.00001 5.50799C4.76358 5.50799 4.57155 5.70001 4.57155 5.93643V8.13416C4.57155 8.37058 4.76358 8.5626 5.00001 8.5626C5.23685 8.5626 5.42847 8.37058 5.42847 8.13416V5.93643ZM7.18428 5.93643C7.18428 5.70001 6.99225 5.50799 6.75582 5.50799C6.51939 5.50799 6.32736 5.70001 6.32736 5.93643V8.13416C6.32736 8.37058 6.51897 8.5626 6.75582 8.5626C6.99266 8.5626 7.18428 8.37058 7.18428 8.13416V5.93643Z" fill="black" />
```

Comment:
At the end of this <path there is still fill="black". I think since we are suing fill=currentColor, we should delete the fill=black